Java/README.md

```diff
@@ -1,46 +1,54 @@
 # Java
 
+If you want Java, you should install OpenJDK.
+"OpenJDK" is the brand name for the modern open-source version of Java.
+That is what you want.
```

Comment:
I would suggest not using this strong language. It should be a choice and you can possibly add that this should be a preferred option.

Reply:
I have toned down the language, and made it more even-handed about OpenJDK vs Oracle JDK.